NSSNamespace/Initial_Bangazon_Site
# removed menu view model

Removed unnecessary view model class

To test:

1. Git fetch
2. Git checkout es-removed-menu-view-model
3. Ensure app works as intended
